script/seed.js

```diff
@@ -2,17 +2,118 @@
 
 const db = require('../server/db')
 const {User} = require('../server/db/models')
+const {Product} = require('../server/db/models')
+const {Category} = require('../server/db/models')
+
```

Comment:
in order for this seed to run "ProductCategory" model needs to exist as an explicitly defined + exported model from models directory - 

`const ProductCategory = db.define('ProductCategory')`

then imported in to this file somewhere here

Reply:
(I think it makes sense to add it in /models/index.js since that's where the relationship is defined using a `through` statement that names the join table)